dnf/cli/commands/repoquery.py

```diff
@@ -413,7 +413,7 @@ def grow_tree(self, level, pkg):
         for reqirepkg in pkg.requires:
             requires.append(str(reqirepkg))
         reqstr = "[" + str(len(requires)) + ": " + ", ".join(requires) + "]"
-        print(spacing + "\_ " + str(pkg) + " " + reqstr)
+        print(spacing + r"\_ " + str(pkg) + " " + reqstr)
```

Comment:
Not sure what you mean -- it's just another invalid escape sequence in a string literal, just like the rest in the commit.

```
$ python --version
Python 3.6.0
$ python -Wd -c '"\_"'
<string>:1: DeprecationWarning: invalid escape sequence \_
```